tck/src/test/java/org/reactivestreams/tck/IdentityProcessorVerificationTest.java

```diff
@@ -115,6 +127,145 @@ public void required_spec104_mustCallOnErrorOnAllItsSubscribersIfItEncountersANo
+            }
+          }
+
+          @Override
+          public void onSubscribe(Subscription s) {
+            this.upstreamSubscription = s;
+          }
+
+          @Override
+          public void onNext(Integer elem) {
+            for (Subscriber<? super Integer> subscriber : subscribers) {
+              try {
+                subscriber.onNext(elem);
+              } catch (Throwable t) {
+                env.flop(t, String.format("Calling onNext on [%s] should not throw! See https://github.com/reactive-streams/reactive-streams-jvm#2.13", subscriber));
+              }
+            }
+          }
+
+          @Override
+          public void onError(Throwable t) {
+            for (Subscriber<? super Integer> subscriber : subscribers) {
+              try {
+                subscriber.onError(t);
+              } catch (Exception ex) {
+                env.flop(ex, String.format("Calling onError on [%s] should not throw! See https://github.com/reactive-streams/reactive-streams-jvm#2.13", subscriber));
+              }
+            }
+          }
+
+          @Override
+          public void onComplete() {
+            for (Subscriber<? super Integer> subscriber : subscribers) {
+              try {
+                subscriber.onComplete();
+              } catch (Exception ex) {
+                env.flop(ex, String.format("Calling onComplete on [%s] should not throw! See https://github.com/reactive-streams/reactive-streams-jvm#2.13", subscriber));
+              }
+            }
+          }
+
+          private Subscription createSubscription(CountDownLatch awaitLatch, final Subscriber<? super Integer> s, final AtomicLong demand) {
+            final MySubscription sub = new MySubscription(awaitLatch, s, demand);
+            subs.add(sub);
+            subscribers.add(s);
+            return sub;
+          }
+
+          final class MySubscription implements Subscription {
+            private final CountDownLatch awaitLatch;
+            private final Subscriber<? super Integer> s;
+            private final AtomicLong demand;
+
+            public MySubscription(CountDownLatch awaitTwoLatch, Subscriber<? super Integer> s, AtomicLong demand) {
+              this.awaitLatch = awaitTwoLatch;
+              this.s = s;
+              this.demand = demand;
+            }
+
+            @Override
+            public void request(final long n) {
+              ex.execute(new Runnable() {
+                @Override
+                public void run() {
+                  if (demand.get() >= 0) {
+                    demand.addAndGet(n);
+                    awaitLatch.countDown();
+                    try {
+                      awaitLatch.await(env.defaultTimeoutMillis(), TimeUnit.MILLISECONDS);
+                      final long d = demand.getAndSet(0);
+                      if (d > 0)  upstreamSubscription.request(d);
```

Comment:
At this point, `upstreamSubscriber` may be null if the `Processor`'s `Subscriber` end hasn't been subscribed to something.